designer/client/src/list/ListEdit.tsx

```diff
@@ -200,6 +213,73 @@ export function ListEdit() {
           errorMessage={errors.title}
         />
 
+        {selectedList.items.length ? (
+          <div className="govuk-form-group">
+            <span className="govuk-label govuk-label--s">
+              {i18n('list.type')}
+            </span>
+            <p className="govuk-body">
+              {`${
+                listTypeRadioItems.find(
+                  (item) => item.value === selectedList.type
+                )?.labelText
+              } values only`}
+            </p>
+          </div>
+        ) : (
+          <div className="govuk-form-group">
+            <fieldset className="govuk-fieldset">
```

Comment:
Mind just wiring up the hint ID for me please?

Hadn't realised **ListEdit.test.tsx** wasn't switched to role tests so this got missed

```suggestion
            <fieldset
              className="govuk-fieldset"
              aria-describedby="list-type-hint"
            >
```